electrum/three_keys/multisig_generator.py

```diff
@@ -0,0 +1,14 @@
+from abc import ABC, abstractmethod
+
+from typing import List
+
+
+class MultisigScriptGenerator(ABC):
```

Comment:
`MultisigScriptGenerator` is not a very good name for the base class, as it's not the pure multisig we're generating. Maybe we should call it `MultiKeyScriptGenerator`

Reply:
done